### PR TITLE
[grpc] update to 1.36.4

### PR DESCRIPTION
**Pull request by youknowone**
~Simple minor update~

### CODE REVIEW

**Review by NancyLi1013 — COMMENTED** [review]
I noticed that the latest version is 1.34.0. Why not update to this version?

**Comment by youknowone** [comment]
I made this while doing other job, so maybe not correctly remembering.
I think 1.34.0 required to fix the vcpkg patches, but I didn't look in it well.

**Comment by NancyLi1013** [comment]
Sorry for my delay.

Do you have any suggestions or problems for the latest version 1.34.0? @youknowone

**Review by ras0219-msft — APPROVED** [review]

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
@youknowone 

Have you tested the feature in this PR?

File: ports/grpc/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "grpc",
-  "version-string": "1.33.1",
-  "port-version": 2,
+  "version-string": "1.33.2",
```
Comment (NancyLi1013) [inline ports/grpc/vcpkg.json:3]:
```suggestion
  "version-semver": "1.33.2",
```
The filed needs to be more proper. Please see https://github.com/microsoft/vcpkg/blob/master/docs/specifications/versioning.md#22-package-versions

File: versions/g-/grpc.json [hunk 2]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "210f3e9f1629abe99deaa4c0061d5e969880b523",
+      "version-string": "1.33.2",
```
Comment (NancyLi1013) [inline versions/g-/grpc.json:5]:
```suggestion
      "version-semver": "1.33.2",
```
The same as above.

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

**Comment by NancyLi1013** [comment]
Hi @youknowone 

Seems there is new release 1.36.1 for `grpc `now.
Would you like to update it to the latest release in this PR?

Thanks.

**Comment by NancyLi1013** [comment]
@youknowone 

Could you please address the review suggestions?

**Comment by youknowone** [comment]
Hi, let me adapt the reviews and update it to grpc 1.36.0

**Comment by youknowone** [comment]
I have a question and a request.

1. what should be the value for `git-tree` in `versions/g-/grpc.json`?
2. To use vcpkg upb, it must be updated to later than 2020-11-xx. Because it changed its cmake files between the version and the latest version of vcpkg, updating upb is not trivial for me. Could anyone help it?

**Comment by PhoebeHui** [comment]
@youknowone, https://github.com/microsoft/vcpkg/pull/17151 would update upb.

**Comment by NancyLi1013** [comment]
`grpc `has been updated to 1.36.4 via PR #17422, which has been merged to maser branch.

So this PR is no longer needed now. Thanks for your contribution. @youknowone

**Pull request closed without merging**